xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -541,12 +541,14 @@ bool CGUIWindowVideoBase::OnSelect(int iItem)
   CFileItemPtr item = m_vecItems->Get(iItem);
 
   std::string path = item->GetPath();
```

Comment:
outside your change but const? same for the CFileItemPtr just above

Reply:
Honestly I thought we'd killed the cfileitemptr already....ffs! It's spread like COVID.